### ISSUE TITLE
[Node 4.0][windows 10] installation error -> MSBUILD error MSB4132  tools version "2.0" is unrecognized

### ISSUE THREAD

**Issue by Clechay** [abbr/deasync#32]
cwd:
``` C:\Users\---\Desktop\--- ```

command:
 ``` λ npm i --save deasync async ```

out:
```
npm WARN package.json ---@1.0.10 No description
npm WARN package.json ---@1.0.10 No repository field.
npm WARN package.json ---@1.0.10 No README data
/
> deasync@0.1.1 install C:\Users\---\Desktop\---\node_modules\deasync
> node ./build.js

C:\Users\---\Desktop\---\node_modules\deasync {git}{hg}                                                                                                                                                                      
{lamb} if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )                                             
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.                                                                                                                           
MSBUILD : error MSB4132: The tools version "2.0" is unrecognized. Available tools versions are "4.0".                                                                                                                                 
gyp ERR! build error                                                                                                                                                                                                                  
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1                                                                                                                            
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)                                                                                                        
gyp ERR! stack     at emitTwo (events.js:87:13)                                                                                                                                                                                       
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)                                                                                                                                                                             
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)                                                                                                                                          
gyp ERR! System Windows_NT 10.0.10240                                                                                                                                                                                                 
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"                                                                             
gyp ERR! cwd C:\Users\---\Desktop\---\node_modules\deasync                                                                                                                                                                   
gyp ERR! node -v v4.0.0                                                                                                                                                                                                               
gyp ERR! node-gyp -v v3.0.1                                                                                                                                                                                                           
gyp ERR! not ok                                                                                                                                                                                                                       
Build failed                                                                                                                                                                                                                          
npm ERR! Windows_NT 10.0.10240                                                                                                                                                                                                        
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "i" "--save" "deasync" "async"                                                                                    
npm ERR! node v4.0.0                                                                                                                                                                                                                  
npm ERR! npm  v2.14.2                                                                                                                                                                                                                 
npm ERR! code ELIFECYCLE                                                                                                                                                                                                              
                                                                                                                                                                                                                                      
npm ERR! deasync@0.1.1 install: `node ./build.js`                                                                                                                                                                                     
npm ERR! Exit status 1                                                                                                                                                                                                                
npm ERR!                                                                                                                                                                                                                              
npm ERR! Failed at the deasync@0.1.1 install script 'node ./build.js'.                                                                                                                                                                
npm ERR! This is most likely a problem with the deasync package,                                                                                                                                                                      
npm ERR! not with npm itself.                                                                                                                                                                                                         
npm ERR! Tell the author that this fails on your system:                                                                                                                                                                              
npm ERR!     node ./build.js                                                                                                                                                                                                          
npm ERR! You can get their info via:                                                                                                                                                                                                  
npm ERR!     npm owner ls deasync                                                                                                                                                                                                     
npm ERR! There is likely additional logging output above.                                                                                                                                                                             
                                                                                                                                                                                                                                      
npm ERR! Please include the following file with any support request:                                                                                                                                                                  
npm ERR!     C:\Users\---\Desktop\---\npm-debug.log              
```

**Comment by nils-glympse** [comment]
I got a similar error on OSX after upgrading to Node 4

```
> deasync@0.0.10 install /Users/joebob/myproject/node_modules/node-horseman/node_modules/deasync
> node ./build.js

Build failed
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE

npm ERR! deasync@0.0.10 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the deasync@0.0.10 install script 'node ./build.js'.
npm ERR! This is most likely a problem with the deasync package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls deasync
npm ERR! There is likely additional logging output above.
```

**Comment by abbr** [comment]
Published 0.1.2 with pre-compiled binaries for node 4. If not working make sure prerequisites in readme are met in order to install from source code.

**Comment by Clechay** [comment]
**Problem solved!**
Thank you for your response :3

solo
```
λ npm i deasync
-
> deasync@0.1.2 install C:\Users\ikilp_000\Desktop\test\node_modules\deasync
> node ./build.js

`win32-x64-node-4.0` exists; testing
Binary is fine; exiting
deasync@0.1.2 node_modules\deasync
├── bindings@1.2.1
└── nan@2.0.9
```

as dependency
```
λ npm i quick-cli
\


> deasync@0.1.2 install C:\Users\ikilp_000\Desktop\test\node_modules\quick-cli\node_modules\deasync
> node ./build.js

`win32-x64-node-4.0` exists; testing
Binary is fine; exiting
quick-cli@1.0.3 node_modules\quick-cli
├── chalk@1.1.1 (escape-string-regexp@1.0.3, supports-color@2.0.0, ansi-styles@2.1.0, has-ansi@2.0.0, strip-ansi@3.0.0)
├── deasync@0.1.2 (bindings@1.2.1, nan@2.0.9)
└── inquirer@0.9.0 (figures@1.3.5, ansi-regex@2.0.0, strip-ansi@3.0.0, cli-width@1.0.1, through@2.3.8, rx-lite@2.5.2, readline2@0.1.1, run-async@0.1.0, lodash@3.10.1)
```
